### ISSUE TITLE
Longer sound fade in/out

### ISSUE THREAD

**Issue by sevmonster** [trynoice/android-app#1095]
### Is your feature request related to a problem?
Similar to the rationale behind #1090, I tend to be completely dead for 30 minutes or so after waking. It would be nice to have either a pre-alarm state with a ramp up to the actual alarm, but an even easier add for now would be to simply...
### Describe the solution you'd like
...Increase the range of the fade sliders. Ten minutes would be good I think.

**Comment by ashutoshgngwr** [comment]
Once #1096 is done, it can be an alarm-specific feature. I plan to add a user setting (Switch) to enable slowly ramping up the master volume when an alarm fires.

**Comment by ashutoshgngwr** [comment]
Instead of a switch, I added an adjustable setting for specifying the ramp-up duration to up to 30 minutes.
